### DISCUSSION TITLE
Change multiSelect key?

### DISCUSSION THREAD

**Discussion by giorgiocaviglia** [prevwong/craft.js#492] [Q&A]
Hi!

First, thanks so much for sharing this project, really awesome :)

I have a question... is there any way to change the key that enables multiSelect (e.g. from Meta to Shift)? I try to change the handlers via options and Editor props but had no luck. Is there any way to do that?

Thanks again!

**Comment by matmanna** [comment]
Is multi-select possible with craft.js? I tried it on the demo on craft.js.org and had no luck 🤔
